CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Coverage Upload Action Changelog
 
+## 2.0.2 (July 26, 2022)
```

Comment:
Since this PR contains both changes we should include this under the same version 2.0.1

Reply:
Should I also change the date?

Reply:
yeah might as well just change it to today's date

Reply:
Good catch. Yeah this should always be the date released. If you put up a PR late then set it to tmw